packit_service/celerizer.py

```diff
@@ -41,9 +42,10 @@ def celery_app(self):
             redis_url = "redis://{host}:{port}/{db}".format(
                 host=redis_host, port=redis_port, db=redis_db
             )
+            postgres_url = f"db+{get_pg_url()}"
```

Comment:
could you please add a comment explaining what the db+ prefix stands for?

Reply:
added